packages/samples/headless-ssr-commerce/components/sort.tsx

```diff
@@ -10,15 +11,19 @@ export default function Sort() {
     return null;
   }
 
-  // TODO: uncomment when KIT-3700: is fixed
-  // const getSortLabel = (criterion: SortCriterion) => {
-  //   switch (criterion.by) {
-  //     case SortBy.Relevancy:
-  //       return 'Relevance';
-  //     case SortBy.Field:
-  //       return criterion.field;
-  //   }
-  // };
+  const getSortLabel = (criterion: SortCriterion) => {
+    switch (criterion.by) {
+      case SortBy.Relevance:
+        return 'Relevance';
+      case SortBy.Fields:
+        return criterion.fields.map((field) =>
+          field
+            ? (field.displayName ??
+              [field.name, field.direction].filter(Boolean).join(' '))
+            : ''
+        );
```

Comment:
Should probably refactor the map callback into a separate function for readability